include/circt/Dialect/SV/SVStatements.td

```diff
@@ -379,6 +379,41 @@ def PAssignOp : SVOp<"passign", [InOutTypeConstraint<"src", "dest">,
   let assemblyFormat = "$dest `,` $src  attr-dict `:` type($src)";
 }
 
+def ForceOp : SVOp<"force", [InOutTypeConstraint<"src", "dest">,
+                                   ProceduralOp]> {
+  let summary = "Force procedural statement";
+  let description = [{
+    A SystemVerilog force procedural statement 'force x = y;'.  These
+    occur in initial, always, task, and function blocks.  
+    A force statement shall override a procedural assignment until
+    a release statement is executed on the variable. 
+    The left-hand side of the assignment can be avariable, a net,
+    a constant bit-select of a vector net, a part-select of a vector 
+    net or a concatenation. It cannot be a memory word or a bit-select 
+    or part-select of a vector variable. See SV Spec 9.3.2.
```

Comment:
9.3.2 is "Parallel Blocks", what is the right section for this?

Reply:
Sorry for the typo, correct section is 10.6.2. Fixed in https://github.com/llvm/circt/commit/e4ece76daf20906211d66b4380b735bef3531f2f.